resources/assets/components/Action/index.js

```diff
@@ -12,6 +12,8 @@ class Action extends React.Component {
   render() {
     const action = this.props.action;
     const campaign = this.props.campaign;
+    const campaignInProgress = parse(campaign.start_date) > Date.now();
```

Comment:
I've been confused about dates recently, but wouldn't a campaign be "in progress" if it's start date is `<` now? (meaning it has already passed?)

Reply:
Oh yeah this is very wrong. 🙃

Reply:
This probably doesn't have anything to do with what it's like in Rogue, but I know that Gambit relies on this date and I think there is something funny with how they "read" the date too and treat it as closed?

Reply:
Updated this in 47b1a2d! Thanks for catching @katiecrane!

Reply:
Maybe we should just add a computed "open" boolean on the campaign response that does this for us so we don't have to risk silly mistakes like this in multiple places. 😂

Reply:
I tested this a little more thoroughly this time and can confirm that it works as expected now: admins can edit actions for open & closed campaigns, but staffers only see the "edit" and "delete" buttons for campaigns that haven't started yet.

Reply:
Oooo I love the computed boolean idea!!